### PR TITLE
Feature/case upload queue

### PR DESCRIPTION
**Pull request by WeiWu2**
### **Changes:**
- My Cases table is now up, previous two table design now merged into one table with two tabs, one for My Pending Cases that contains columns of Case Number, Uploaded On, View PDF, Status and Cancel, and the other tab, My Approved Cases that contains columns of Case Number, View PDF, and Status.

### **Known Bugs/Still Working On**:
- View PDF
- Uploading a case does not auto refresh the table(have to manual refresh to get updated pending cases)
- Cancel button does work, but still working on having a confirmation notice to confirm cancelation (also does not auto refresh the table)
- Filter icon/button
- Font size change of tab title not tied to actual tab, its tied to tab click(clicking on the same tab multiple times will change font)

### CODE REVIEW

File: src/components/pages/MyCases/MyCases.js [hunk 1]
```diff
@@ -1,124 +1,340 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import { trackPromise } from 'react-promise-tracker';
```
Comment (MichelleCCodes) [inline src/components/pages/MyCases/MyCases.js:3]: great use of this component in mycases!

**Review by MichelleCCodes — APPROVED** [review]
These changes look good, and I'm excited to go in and add to it!

File: src/components/pages/MyCases/MyCases.js (hunk 1 above)
Comment (paulstgermain) [reply src/components/pages/MyCases/MyCases.js:3]: Definitely seconding this!

**Review by paulstgermain — APPROVED** [review]
Wow, excellent refactoring of this feature, I'm excited to see this come together and work!

**Pull request merged**